analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+                              'prs': {'count': 0, 'age': 0},
+                              'p0': {'count': 0, 'age': 0},
+                              'p1': {'count': 0, 'age': 0},
+                              'p2+': {'count': 0, 'age': 0},
+                              'no_priority_label': {'count': 0, 'age': 0},
+                              'fr_question': {'count': 0, 'age': 0}}
+
+            analysis[repo]['issues']['count'] = data_decode[repo][
+                                                  'open_issues_count']
+            analysis[repo]['prs']['count'] = data_decode[repo]['prs'][
+                                               'open_pr_count']
+
+            # if there are prs, calculate their mean age
+            if analysis[repo]['prs']['count'] > 0:
+                analysis[repo]['prs']['age'] = data_decode[repo]['prs'][
+                    'pr_aggregate_age']/analysis[repo]['prs']['count']
+
+            for issue in data_decode[repo]['issues']:
+                created_date = dt.strptime(
+                    data_decode[repo]['issues'][issue]['created'],
+                    "%Y-%m-%dT%H:%M:%S+00:00")
+
+                # it is invalid for an issue to carry multiple priority labels
+                # so count the most severe p0 THEN p1 THEN p2+ THEN fr_question
+
+                if any(x in data_decode[repo]['issues'][issue]['labels']
+                       for x in p0_labels):
+                    analysis[repo]['p0']['count'] += 1
+                    analysis[repo]['p0']['age'] += (dt.utcnow() -
+                                                    created_date).days
+
+                elif any(y in data_decode[repo]['issues'][issue]['labels']
+                         for y in p1_labels):
+                    analysis[repo]['p1']['count'] += 1
+                    analysis[repo]['p1']['age'] += (dt.utcnow() -
+                                                    created_date).days
+
+                elif any(z in data_decode[repo]['issues'][issue]['labels']
+                         for z in p2_labels):
+                    analysis[repo]['p2+']['count'] += 1
+                    analysis[repo]['p2+']['age'] += (dt.utcnow() -
+                                                     created_date).days
+
+                elif any(x in data_decode[repo]['issues'][issue]['labels']
+                         for x in fr_question):
+                    analysis[repo]['fr_question']['count'] += 1
+                    analysis[repo]['fr_question']['age'] += (
+                        dt.utcnow() - created_date).days
+
+                else:
+                    analysis[repo]['no_priority_label']['count'] += 1
+                    analysis[repo]['no_priority_label']['age'] += (
+                        dt.utcnow() - created_date).days
+
+            if analysis[repo]['p0']['count'] > 0:
+                analysis[repo]['p0']['age'] = analysis[repo]['p0'][
+                    'age']/analysis[repo]['p0']['count']
```

Comment:
Also, this can be more easily written:

```python
analysis[repo]['p0']['age'] /= analysis[repo]['p0']['count']
```

I recommend doing this everywhere that you currently use `x = x / y`, since it will make it fit on one line.